.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+      "liveServer.settings.port": 5501
+}
```

Comment:
As this file is not part of this project, you could add its path in the `.gitignore` file which won't add it here in the GitHub.